openfl-tutorials/interactive_api/Pytorch_Market_Re-ID/envoy_folder/one/market_shard_descriptor.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Market shard descriptor."""
+
+from pathlib import Path
+import re
+from logging import getLogger
+
+logger = getLogger(__name__)
+
+import numpy as np
+from PIL import Image
+
+from openfl.interface.interactive_api.shard_descriptor import ShardDescriptor
+
+
+DATAPATH = list(Path.cwd().parents[2].rglob('**/Market'))[0]    # parent directory of project
```

Comment:
`IndexError: list index out of range`

Reply:
added comment below to download data and put to directory previously

Reply:
Can data be downloaded automatically?

Reply:
No, I found two sources: google-drive and kaggle, but both of them are pretty hard to download automatically